content/en/docs/registry-support.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Registry support"
+description: "A list of registries that support Notary Project OCI signatures"
+type: docs
+weight: 9
+---
+
+The following registries, listed in alphabetical order, support Notary Project OCI signatures:
+
+- [Amazon Elastic Container Registry](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html)
+- [Azure Container Registry](https://learn.microsoft.com/azure/container-registry/?wt.mc_id=azurelearn_inproduct_oss_notaryproject)
+- [Docker Hub](https://hub.docker.com)
+- [GitHub Container Registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry)
+- [Harbor](https://goharbor.io/)
+- [JFrog Artifactory](https://jfrog.com/artifactory/)
+- [ORAS Distribution Registry](https://github.com/oras-project/distribution/pkgs/container/registry/64589674?tag=v1.0.0-rc.4)
```

Comment:
ORAS Distribution will be archived soon and will not be actively maintained anymore. Maybe we could remove it.

[Distribution/Distribution v3)](https://github.com/distribution/distribution) might be a registry that is compliant with Notation but I need to double check.

Reply:
I tested notation with [Distribution/Distribution (v3)](https://github.com/distribution/distribution) registry and it works with using the tag schema. We can replace the [ORAS Distribution Registry](https://github.com/oras-project/distribution/pkgs/container/registry/64589674?tag=v1.0.0-rc.4) with [Distribution/Distribution](https://github.com/distribution/distribution).

Reply:
Updated. For the registry used in quick start, maybe we can consider docker as it is popular and users do not need to install something locally, which can be convenient. But this is another topic out of the scope of this PR.